### PR TITLE
Fix README file

### PR DESCRIPTION
**Pull request by edoardob90**
Remove incorrect line about updating conda environment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,12 +67,6 @@ Finally, launch JupyterLab with
 jupyter lab
 ```
 
-To update the existing environment, run
-
-```console
-conda env update -f environment.yml
```
Comment (yakutovicha) [inline README.md:73]: Maybe instead of deleting it, you could update it to:
```bash
conda env update -f binder/environment.yml
```

**Review by yakutovicha — COMMENTED** [review]
I'm not sure if the PR is completed, but here is my quick review.

File: README.md (hunk 1 above)
Comment (edoardob90) [reply README.md:73]: It's already there:
https://github.com/empa-scientific-it/python-tutorial/blob/f7149925eddcdbcd875a42c6a7a1c47b35e9a1a3/README.md?plain=1#L56-L60

Comment (edoardob90) [reply README.md:73]: Incidentally: Conda here is barely useful. All the packages are installed with `pip`, and using Conda slows down quite a lot the building of the Docker image. If you try it locally, you'll see why.

Comment (edoardob90) [reply README.md:73]: But as the smart people say: "if it ain't broken, don't fix it".

**Review by yakutovicha — APPROVED** [review]
go ahead 👍

**Comment by edoardob90** [comment]
Everything's green now.

**Review by yakutovicha — APPROVED** [review]
One more green ✅

**Pull request merged**